tests/timeseries_util.erl

```diff
@@ -41,101 +59,91 @@
 -define(MAXFLOAT,            math:pow(2, 63)).
 -define(MULTIPLECLUSTERSIZE, 3).
 
-confirm_create(ClusterType, DDL, Expected) ->
+-spec(confirm_create(atom(), string()) -> {ok, string()} | term()).
+confirm_create(ClusterType, DDL) ->
 
-    [Node | _] =build_cluster(ClusterType),
+    [Node | _] = build_cluster(ClusterType),
 
     Props = io_lib:format("{\\\"props\\\": {\\\"n_val\\\": 3, \\\"table_def\\\": \\\"~s\\\"}}", [DDL]),
-    Got = rt:admin(Node, ["bucket-type", "create", get_bucket(), lists:flatten(Props)]),
-    ?assertEqual(Expected, Got),
-
-    pass.
+    rt:admin(Node, ["bucket-type", "create", get_bucket(), lists:flatten(Props)]).
 
-confirm_activate(ClusterType, DDL, Expected) ->
+confirm_activate(ClusterType, DDL) ->
 
     [Node | Rest] = build_cluster(ClusterType),
     ok = maybe_stop_a_node(ClusterType, Rest),
     {ok, _} = create_bucket(Node, DDL, 3),
-    Got     = activate_bucket(Node, DDL),
-    ?assertEqual(Expected, Got),
-
-    pass.
+    activate_bucket(Node, DDL).
 
-confirm_put(ClusterType, TestType, DDL, Obj, Expected) ->
+confirm_put(ClusterType, TestType, DDL, Obj) ->
```

Comment:
Speaking of names, I would not ordinarily expect a `put` function to build a cluster. I'd consider making these tests more "functional"; instead of passing in ClusterType, something like:

```
confirm_put(activate_bt(create_bt(build_cluster(ClusterType), DDL, 3), DDL), Obj)
```

And then the failure mode without creation/activation would be
```
confirm_put(build_cluster(ClusterType), Obj)
```

which would also work better if you wanted to run multiple puts in the same test